Phoenix/hello/lib/hello_web/router.ex

```diff
@@ -18,6 +19,15 @@ defmodule HelloWeb.Router do
     pipe_through :browser
 
     get "/", PageController, :index
+    get "/hello", HelloController, :index
+    get "/hello/:messenger", HelloController, :show
+    resources "/users", UserController
```

Comment:
@samu975 What do resources mean?

Reply:
resouces is a macro. We use the resouces macro to add common routes and implement common actions like create, read, update and delete (crud)